modules/friendlytag.js

```diff
@@ -570,6 +578,24 @@ Twinkle.tag.updateSortOrder = function(e) {
 	if (alreadyPresentTags) {
 		alreadyPresentTags.forEach(generateLinks);
 	}
+
+	var statusElement = document.getElementById('tw-tag-status');
+	$('[name=articleTags], [name=alreadyPresentArticleTags]').click(function() {
+		if (this.name === 'articleTags') {
+			if (this.checked) Twinkle.tag.status.numAdded++;
+			else Twinkle.tag.status.numAdded--;
+		} else if (this.name === 'alreadyPresentArticleTags') {
+			if (this.checked) Twinkle.tag.status.numRemoved--;
+			else Twinkle.tag.status.numRemoved++;
+		}
+
+		// update statusElement text
+		var firstPart = 'Added ' + Twinkle.tag.status.numAdded + ' tag' + (Twinkle.tag.status.numAdded > 1 ? 's' : '');
```

Comment:
Past tense seems weird.  Could also prefix the whole thang with "Will " to make "Will add 3 add remove 2" for a complete-ish sentence, but that seems excessive.
```suggestion
		var firstPart = 'Adding ' + Twinkle.tag.status.numAdded + ' tag' + (Twinkle.tag.status.numAdded > 1 ? 's' : '');
```